doc/crud.md

```diff
@@ -40,6 +55,10 @@ CDTDatastore *ds = [manager datastoreNamed:@"my_datastore"
 CDTDatastore *ds2 = [manager datastoreNamed:@"other_datastore"
                                       error:&error];
 ```
+```swift
+let ds = try manager.datastoreNamed("my_datastore")
+let ds2 = try manager.datastoreNamed("other_datastore")
```

Comment:
@rhyshort: I do see your point though about not including the `catch`.  The comparable Objective C example does not show any error handling so why would we include it in the Swift example.  So disregard my previous comment (https://github.com/cloudant/CDTDatastore/pull/210/files#r45383243), I think it is fine to leave the `catch` out.